net-dialup/radiusclient/files/radiusclient-0.3.2-BJA-cross.diff

```diff
@@ -0,0 +1,67 @@
+--- configure.in	2002-02-03 14:51:40.000000000 +0100
```

Comment:
Aye, freecode is basically dead. I hope the project has some upstream these days…